Skills.md

```diff
@@ -1,61 +1,80 @@
 # Skill/Attribute Challenges
+You may have a maximum of 5 gems in a sphere.
 
-Challenges are represented by rolling dice either over, or under a given threshold for multiple attributes (elements).
-This gives the GM a lot of flexibility around how skill challenges occur. For any particular skill challenge,
-the GM picks 3 attributes (attributes may be repeated). The player must then roll a die, and get under the threshold
-for each attribute. The threshold is the 2x number of gems in the spheres matching the element in question, +1 for each gem
-of the same element. The die rolled
-depends on the number of gems owned of the matching element. Thresholds can be adjusted by the GM to demonstrate difficulty.
+## Challenge Rules
+Challenges are handled by rolling three dice, associated with three elements. There is one major element, and two minor.
+A success requires rolling greater than a threshold on the major element, and a least one of the minor elements. A failure
+on only one of the minor elements results in a minor negative side effect, the details of which are left up to the GM,
+but where possible associated with the element that failed.
```

Comment:
Any thoughts on a major failure but two minor successes? Conditions/consequences for a glitch?

Reply:
Not overly yet, though I think it could have some interesting ideas.